runtime/lib/Menu.gp

```diff
@@ -4,14 +4,17 @@ defineClass Menu morph label target items reverseCall selection returnFocus
 
```

Comment:
Need to discuss whether bigger is better for the whole interface

Reply:
This can be deleted if someone truly doesn't like it, for now I'm committing to see how it goes